src/App.js

```diff
@@ -1,5 +1,37 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+import Car from './models/Car.js';
+import {
+  validateCarName,
+  validateCarNumber,
+  validateRoundNum,
+} from './validators/validator.js';
+import MESSEGE from './constants/messeges.js';
+import { getRaceresult, getWinners } from './game/race.js';
+
 class App {
-  async play() {}
+  async play() {
+    // 게임 시작
+    const carNames = await MissionUtils.Console.readLineAsync(
+      MESSEGE.carNamesInput,
+    );
+
+    let carArray = carNames.split(',').map(temp => temp.trim());
+    validateCarNumber(carArray);
+    validateCarName(carArray);
+    carArray = carArray.map(carName => new Car(carName));
```

Comment:
로직을 명확하게 작성해주셔서 무리 없이 읽을 수 있었습니다 🙂
다만 크게 두 가지 개선해볼 수 있을 것 같다는 생각이 들었습니다.
1) 변수명 개션
* temp -> carName : 더 명확한 단어로 표현하면 더 좋을 것 같아요!
* carArray -> cars : 자료형을 변수명에 안 쓸 수 있으면 안 쓰는 게 좋다고 하더라구요! 예를 들어 이 배열이 set 타입으로 바뀐다면 변수명도 바꿔줘야 하는 번거로움이 있으니까요. 그래서 복수형으로 대체할 수 있을 것 같아요!
2) let 사용피하기
let을 쓰면 이후의 로직에서 해당 변수가 언제 변할지 몰라 읽는 입장에서는 약간의 어려움을 겪을 수 있을 것 같아요!

```suggestion
    const carNamesInput = await MissionUtils.Console.readLineAsync(
      MESSEGE.carNamesInput,
    );

    const carNames = carNamesInput.split(',').map(carName => carName.trim());
    validateCarNumber(carNames);
    validateCarName(carNames);
    const cars = carNames.map(carName => new Car(carName));
```